scripts/index.js

```diff
@@ -0,0 +1,154 @@
+    rainy: "https://images.unsplash.com/photo-1610741083757-1ae88e1a17f7?ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&ixlib=rb-1.2.1&auto=format&fit=crop&w=1050&q=80",
+    snowy: "https://images.unsplash.com/photo-1518873890627-d4b177c06e51?ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&ixlib=rb-1.2.1&auto=format&fit=crop&w=1050&q=80"
+}
+
+const changeTempColor = () => {
+    const tempValue = document.querySelector("#tempValue");
+    const tempDisplay = document.querySelector("#tempDisplay");
+
+    if(state.temperature >= 80){
+        tempDisplay.style.color = weather_info.hot.fontColor
+        tempValue.style.color = weather_info.hot.fontColor
+    }else if(state.temperature <= 79 && state.temperature >=70){
+        tempDisplay.style.color = weather_info.warm.fontColor
+        tempValue.style.color = weather_info.warm.fontColor
+    }else if(state.temperature <= 69 && state.temperature >=60){
+        tempDisplay.style.color = weather_info.cool.fontColor
+        tempValue.style.color = weather_info.cool.fontColor
+    }else if(state.temperature < 59){
+        tempDisplay.style.color = weather_info.freezing.fontColor
+        tempValue.style.color = weather_info.freezing.fontColor
+    }
+}
+
+const changeLandscape = () => {
+    const lanscapeContent = document.querySelector("#lanscapeContent");
+    const landscapeContainer = document.querySelector("#landscapeContainer");
+    if(state.temperature >= 80){
+        landscapeContainer.style.color = weather_info.hot.color
+        lanscapeContent.textContent = `${weather_info.hot.emojis}`;
+
+    }else if(state.temperature <= 79 && state.temperature >=70){
+        landscapeContainer.style.color = weather_info.warm.color
+        lanscapeContent.textContent = `${weather_info.warm.emojis}`;
+
+    }else if(state.temperature <= 69 && state.temperature >=60){
+        landscapeContainer.style.color = weather_info.cool.color
+        lanscapeContent.textContent = `${weather_info.cool.emojis}`;
+
+    }else if(state.temperature < 59){
+        landscapeContainer.style.color = weather_info.freezing.color
+        lanscapeContent.textContent = `${weather_info.freezing.emojis}`;
+    }
+}
+
+const changeTempValue = () => {
+    const tempValue = document.querySelector("#tempValue");
+    tempValue.textContent = `Temperature: ${state.temperature}F°`;
+    const tempDisplay = document.querySelector("#tempDisplay");
+    tempDisplay.textContent = `${state.temperature}F°`;
+}
+
+const decreaseTemp = (event) => {
+    state.temperature -= 1;
+    changeTempValue()
+    changeTempColor()
+    changeLandscape()
+};
+
+const increaseTemp = (event) => {
+    state.temperature += 1;
+    changeTempValue()
+    changeTempColor()
+    changeLandscape()
+};
+
+const changeSky = (event) => {
+    const skyContent = document.querySelector("#skyContent");
```

Comment:
If looking an element up by id, prefer `getElementById`. `querySelector` with an an `#id` would be more appropriate if we need to get a _child_ of that element, like `document.querySelector("#someId p")`.